charts/ccsm-helm/charts/zeebe/templates/statefulset.yaml

```diff
@@ -172,6 +174,7 @@ spec:
       tolerations:
 {{ toYaml . | indent 8 }}
 {{- end }}
+{{- if .Values.persistence.enabled }}
```

Comment:
:thought_balloon: Do you think it might make sense to call it  `pvcEnabled`? Since the other are also called with prefix `pvc`